vyper/ast/grammar.lark

```diff
@@ -283,8 +271,22 @@ _IN: "in"
        | "+"  power -> uadd
        | "-"  power -> usub
        | "~"  power -> invert
-?power: atom
-      | power _POW  atom -> pow
+
+// TODO: add factor rule
+?power: external_call
+      | external_call _POW  power -> pow
+
+?external_call: ("extcall" | "staticcall")? atom_expr
```

Comment:
should these labels be translated into more specific keywords?

Reply:
also, should this syntax be optional to produce an external call rule?

Reply:
It has the same precedence as await. See the lark grammar linked from the PR description